docs/developers/api-upload-data.md

```diff
@@ -231,28 +229,28 @@ def action():
     streamer.add_excel_file('~/Downloads/SomeRandomDocument.xlsx')
 ```
 
-## Import Data Already Ingested into Elastic.
+## Import Data Already Ingested into OpenSearch.
 
-You may have other mechanism to ingest data into Elastic, like an ELK stack or
-some manual scripts that ingest the data. Since the data is already in Elastic
+You may have other mechanism to ingest data into OpenSearch, like an ELK stack or
```

Comment:
Is that ELK stack then? Or should we rewrite that sentence?

Reply:
There is not another term for it. Kibana -> Dashboards. I opt to keep it here seeing people can relate to ELK.

Reply:
SGTM